docs/setupDevEnvironmentUbuntu20.04.md

```diff
@@ -0,0 +1,161 @@
+# Setup development environment on Ubuntu
+
+*Tested on Ubuntu 20.04 LTS (Focal Fossa)*
+
+These instructions will setup a complete development environment for Transition on a new installation of Ubuntu (like a virtual machine). Some of these instructions are not necessary to simply run the application.
+
+## Prerequisites
+* yarn
+```
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list 
+sudo apt update && sudo apt install yarn
+```
+* Cargo
+```
+sudo apt install cargo
+```
+
+## Update ubuntu
+```
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get dist-upgrade
+```
+
+## Install dependencies
+```
+sudo apt-get install git ssh curl wget gcc g++ cmake \
+libstxxl-dev libxml2-dev libsparsehash-dev libbz2-dev zlib1g-dev libzip-dev libgomp1 \
+pkg-config libgdal-dev libtbb-dev psmisc build-essential postgresql postgis \
+clang libboost-all-dev libexpat1-dev libjsoncpp-dev libncurses5-dev lua5.3 liblua5.3-dev \
+powerline fonts-powerline zsh capnproto libcapnp-dev postgresql-12-postgis-3 \
+postgresql-12-postgis-3-scripts rustc
```

Comment:
maybe replace the version specific sript package with the meta one: "postgresql-postgis-scripts"